src/nm_mon_daemon.c

```diff
@@ -431,6 +429,8 @@ void nm_mon_loop(void)
     }
 
 #if defined (NM_OS_LINUX)
```

Comment:
We should remove #if defined (NM_OS_LINUX) if using atexit(3) because it exists in FreeBSD.
This defines uses only for at_exit calls.